OpenRA.Mods.Common/Traits/World/EditorResourceLayer.cs

```diff
@@ -58,6 +60,27 @@ public void WorldLoaded(World w, WorldRenderer wr)
 
 			foreach (var cell in Map.AllCells)
 				UpdateCell(cell);
+
+			// Build the sprite layer dictionary for rendering resources
+			// All resources that have the same palette must also share a sheet and blend mode
```

Comment:
> ClearSides FindClearSides(ResourceType t, CPos p)

is something our dupFinder (see https://ci.appveyor.com/project/OpenRA/openra/build/1.0.4556/artifacts for example) spits out, but they are not too easy to unduplicate. They look identical from static source code analysis, but the `CellContains` is completely different.